socorro/external/es/super_search_fields.py

```diff
@@ -1448,6 +1448,34 @@ def number_field(
         "query_type": "enum",
         "storage_mapping": {"type": "string"},
     },
+    "crashing_thread_id": {
+        "data_validation_type": "int",
+        "description": "",
+        "form_field_choices": [],
+        "has_full_version": False,
+        "in_database_name": "crashing_thread_id",
+        "is_exposed": True,
+        "is_returned": True,
+        "name": "crashing_thread_id",
+        "namespace": "processed_crash",
+        "permissions_needed": [],
+        "query_type": "number",
+        "storage_mapping": {"type": "integer"},
+    },
+    "crashing_thread_name": {
```

Comment:
I think this should probably be implemented as a keyword--not a string. Strings get analyzed when indexed, but keywords don't. I think you should use `keyword_field`. Here's an example:

https://github.com/mozilla-services/socorro/blob/6e7aadc03a2ba5315a978fac433285dde8758af3/socorro/external/es/super_search_fields.py#L1468-L1475

Having said that, I don't exactly have a list of thread name example values in front of me, so maybe keyword isn't a good fit. We'd have to test it out.